x-pack/test_serverless/functional/test_suites/observability/dataset_quality/dataset_quality_table.ts

```diff
@@ -19,42 +25,79 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
     'svlCommonPage',
   ]);
   const synthtrace = getService('svlLogsSynthtraceClient');
-  const testSubjects = getService('testSubjects');
-  const retry = getService('retry');
   const to = '2024-01-01T12:00:00.000Z';
+  const apacheAccessDatasetName = 'apache.access';
+  const apacheAccessDatasetHumanName = 'Apache access logs';
+  const pkg = {
+    name: 'apache',
+    version: '1.14.0',
+  };
 
   describe('Dataset quality table', function () {
-    this.tags(['failsOnMKI']); // Failing https://github.com/elastic/kibana/issues/183495
-
     before(async () => {
-      await synthtrace.index(getInitialTestLogs({ to, count: 4 }));
+      // Install Integration and ingest logs for it
+      await PageObjects.observabilityLogsExplorer.installPackage(pkg);
+      // Ingest basic logs
+      await synthtrace.index([
+        // Ingest basic logs
+        getInitialTestLogs({ to, count: 4 }),
+        // Ingest Degraded Logs
+        getLogsForDataset({
+          to: new Date().toISOString(),
+          count: 1,
+          dataset: datasetNames[2],
```

Comment:
> [!WARNING]  
> This might not be in the scope of this PR, but it is really difficult to understand into which dataset we are ingesting logs, using a map of these names could make it easier to debug.

Reply:
I agree, as the refactor was huge, i kept the same code in some places, promise to fix this soon 👍🏼